docs/premium-features/custom-types-and-actions.md

```diff
@@ -6,25 +6,27 @@ slug: /custom-types-and-actions
 
 # Custom Types & Actions
 
-Custom Types & Actions is an enhanced feature that gives you full control over your Data Transfer Objects (DTOs) and APIs. It allows you to automatically generate high-quality, efficient code for your REST and/or GraphQL APIs.
+Custom Types & Actions is an enhanced feature that gives you full control over your Data Transfer Objects (DTOs) and APIs. It allows you to automatically generate high-quality, efficient code for both the default CRUD operations and your custom APIs that utilize custom DTOs, all within your REST and/or GraphQL APIs.
 
-Custom Types & Actions let you define custom data structures, create API endpoints, and manage the relationships between them, all within the Amplication UI.
+Custom Types & Actions let you define custom data structures, create API endpoints, and manage the relationships between them, all within the Amplication UI. These changes are automatically translated into production-ready code in your repository, ensuring the best quality and ,efficiency.
 
 :::note
-The Custom Types & Actions feature is available for Amplication's **Enterprise** plan.
+The Custom Types & Actions feature is available for Amplication's **Enterprise** plan, including trial plans.
 :::
 
```

Comment:
I think we need to emphasize and clarify somewhere at the beginning of the doc, that Actions = APIs, and Types = DTOs, and primitives like String, Integer etc.
I noticed you mentioned it regarding the Actions, but only later in the doc, and I believe it should be somewhere at its beginning.